drivers/overlay/ov_network.go

```diff
@@ -260,6 +262,20 @@ func (n *network) Index() uint64 {
 
 func (n *network) SetIndex(index uint64) {
 	n.dbIndex = index
+	n.dbExists = true
+}
+
+func (n *network) Exists() bool {
+	return n.dbExists
+}
+
+func (n *network) SetValue(value []byte) error {
+	var vni uint32
+	err := json.Unmarshal(value, vni)
```

Comment:
You need to pass the pointer to `vni` to `Unmarshal`. Otherwise you are only setting the zero value in the network structure